scripts/initscripts/snapd.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Starts and stops a single snapd instance on this system
+Documentation=https://github.com/intelsdi-x/snap
+Wants=network-online.target
+After=network-online.target
```

Comment:
Currently my systemd script is simply After=network.target. Should I switch to After=network-online.target?